doc-deploy-dev/action.yml

```
@@ -1,43 +1,72 @@
-name: "Documentation deploy development version."
-description: "Deploy development documentation artifacts to desired repository branch."
+name: >
+  Doc deploy dev action
+
+description: >
+  This action deploys the desired HTML documentation artifact containing the
+  development version of a library to the specified branch of a repository. By
+  default, the ``gh-pages`` branch of the current repository is assumed.
 
 inputs:
+
+  # Required inputs
+
+  cname:
+    description: >
+      The canonical name (CNAME) containing the documentation.
+    required: true
+    type: string
+
+  token:
+    description: >
+      Required password, key or token with right credentials to deploy the
+      documentation. If deploying to the current repository, the ``${{ secrets.GITHUB_TOKEN }}``
+      is enough. For workflows deploying to other repositories, use a different token.
```

```suggestion
      Required password, key, or token with the correct credentials for deploying the
      documentation. If deploying to the current repository, the ``${{ secrets.GITHUB_TOKEN }}``
      token is enough. For workflows deploying to other repositories, use a different token.
```
How would someone know what "different token" to use?